gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/PageServiceImpl.java

```diff
@@ -777,10 +782,6 @@ private PageEntity createPage(String apiId, NewPageEntity newPageEntity, String
                 checkFolderConsistency(newPageEntity);
             }
 
-            if (SYSTEM_FOLDER.equals(newPageType)) {
-                assertNoSystemFolderExists(environmentId, apiId);
```

Comment:
this prevents someone to create a system folder using the management API, I think this test should remain

Reply:
As discussed together, this exception thrown during import could, in some edge cases, block the user from promoting.
So we changed this behavior to an ERROR message, saying the it will update the existing system folder instead of creating a new one.